dir.common.props

```diff
@@ -65,7 +65,7 @@
 
   <PropertyGroup>
     <!-- Central place to set the versions of all nuget packages produced in the repo -->
-    <PackageVersion Condition="'$(PackageVersion)' == ''">3.1.13</PackageVersion>
+    <PackageVersion Condition="'$(PackageVersion)' == ''">3.1.15</PackageVersion>
```

Comment:
yes we need to change this https://github.com/dotnet/coreclr/pull/28151/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67R5 as well to 3.1.15

Reply:
fixed